priv/repo/migrations/20250905102314_change_pwd_to_physically_handicapped_in_demographic_profile.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Dbservice.Repo.Migrations.ChangePwdToPhysicallyHandicappedInDemographicProfile do
+  use Ecto.Migration
+
+  def change do
+    alter table(:demographic_profile) do
+      remove :pwd
+      add :physically_handicapped, :boolean, default: false
+    end
+  end
+end
```

Comment:
We don’t need this since it hasn’t been merged to main or release (so the migrations haven’t run on staging or prod). You can simply edit the previous migration, clean up your local db, and re-run the migrations locally to verify the changes